packages/web/modals/ibc-transfer.tsx

```diff
@@ -2,6 +2,7 @@ import { FunctionComponent, useState } from "react";
 import { observer } from "mobx-react-lite";
 import { useStore } from "../stores";
 import { Transfer } from "../components/complex/transfer";
+import { IbcStatus } from "../../stores/src/queries-external";
```

Comment:
This is the culprit of the build error! Was totally stumping me as well. I think there's differing tsconfig for building next.js apps that are not present in our stores package and other packages. This seems unused, so it can be removed. But in the future make sure you import from `@osmosis-labs/stores` instead of by file path in the future. We use a yarn workspace/monorepo setup that resolves the above name into the build product of the packages in the workspace, instead of referencing the package contents directly.

Reply:
It was unused, but good to know